src/views/v2/commerce/checkout/purchaser-info.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Tickets Commerce: Checkout Purchaser Info.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/checkout/purchaser-info.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since TBD
+ *
+ * @version TBD
+ *
+ * @var \Tribe__Template $this [Global] Template object.
+ * @var array[] $items [Global] List of Items on the cart to be checked out.
+ * @var bool $must_login Global] Whether login is required to buy tickets or not.
+ */
+
+if ( is_user_logged_in() || $must_login || empty( $items ) ) {
+	return;
+}
+
+$title = __( 'Purchaser info', 'event-tickets' );
+?>
+
+<div class="tribe-tickets__form tribe-tickets__commerce-checkout-purchaser-info-wrapper">
+	<h4 class="tribe-common-h5"><?php echo esc_html( $title ); ?></h4>
```

Comment:
Maybe do the `esc_html_e()` here of the localized string so we avoid having a variable there, which is not being manipulated or anything.